pkgs/applications/misc/spacefm/default.nix

```diff
@@ -4,7 +4,7 @@
 
 stdenv.mkDerivation rec {
   pname = "spacefm";
-  version = "1.0.6";
+  version = "1.0.6-unstable-2021-02-09";
```

Comment:
```suggestion
  version = "unstable-2021-02-09";
```

Reply:
Since there is no version bump but only a patch added, why do we need to change the version at all?

Reply:
Also this version is not a valid tag and breaks fetchFromGitHub.

Reply:
> Since there is no version bump but only a patch added, why do we need to change the version at all?

Yeah, right. We can just drop the version bump all together.

Reply:
I thought this would be needed to invalidate the existing cache entry? How does nix(-build) determine if it can fetch a binary instead of building it locally?

Reply:
BTW, I have removed the version change from the commit.